features/sync/current_branch/feature_branch/strategy_override/sync_strategy_override.feature

```diff
@@ -0,0 +1,54 @@
+Feature: override the configured sync-feature-strategy
+
+  Background:
+    Given a Git repo with origin
+    And the branches
+      | NAME    | TYPE    | PARENT | LOCATIONS     |
+      | feature | feature | main   | local, origin |
+    And Git setting "git-town.sync-feature-strategy" is "merge"
```

Comment:
Maybe move this line down to above line 16 so that the currently set `sync-feature-strategy` and the override are close to each other? This would make it easier to see which strategy is set and what it's overridden to.